ush/module-setup.sh

```diff
@@ -22,6 +22,13 @@ elif [[ $MACHINE_ID = orion* ]] ; then
     fi
     module purge
 
+elif [[ $MACHINE_ID = hercules* ]] ; then
+    # We are on Orion
```

Comment:
should this be "on Hercules"?

Reply:
@TingLei-NOAA Thanks for catching this! I will change it as suggested.